src/index.js

```diff
@@ -1,6 +1,11 @@
-import displayTasks from './modules/display.js';
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Linting is a great tool that helps finds errors in your code so that you may fix them, so I think it is better to avoid disabling them